about/exploring.md

```diff
@@ -40,12 +40,16 @@ In other words, WPE is designed to be _built_ and optimized for your embedded de
 ### Easy ways to explore WPE
 All of this said, we believe that enabling developers to explore the space is important for a whole lot of reasons.  Therefore, we've tried to make it easy for developers who aren't comfortable building entire embedded systems to get their hands on _something_ where they can begin exploring and learning useful things.
 
-* <span class="btn btn-primary text-light">Install it from your distribution</span>: WPE WebKit is available in some Linux distributions including [Debian](https://packages.debian.org/search?searchon=sourcenames&keywords=wpewebkit), [Ubuntu](https://packages.ubuntu.com/search?keywords=wpewebkit&searchon=sourcenames&suite=all&section=all) and [Arch Linux](https://archlinux.org/packages/extra/x86_64/wpewebkit/). These packages are not just a quick and simple way to test WPE but they also come with all the development files and documentation necessary to build and test software that uses this web engine.
+* __Install it from your GNU/Linux OS distribution__: These packages are not just a quick and simple way to test WPE but they also come with all the development files and documentation necessary to build and test software that uses this web engine.  WPE WebKit is available in several Linux distributions including:
+  * [Debian](https://packages.debian.org/search?searchon=sourcenames&keywords=wpewebkit)
+  * [Ubuntu](https://packages.ubuntu.com/search?keywords=wpewebkit&searchon=sourcenames&suite=all&section=all)
+  * [Raspbian](https://archive.raspbian.org/raspbian/pool/main/w/wpewebkit/)
+  * [Arch Linux](https://archlinux.org/packages/extra/x86_64/wpewebkit/)
 
 * <span class="btn btn-primary text-light">[Install with Flatpak]({{'/about/flatpak.html' | url}})</span>: If you have support for Flatpak on your current hardware and operating system, and your graphics drivers are supported, there are various options to try WPE through Flatpak.  This can be a very easy way to let you explore WPE, check the features and specifications supported by WPE, or just want to experiment and learn more about what some of the differences with a full browser are.
```

Comment:
While at it, can we change this link to be a regular one instead of an awkward-looking button-like thing? It should be enough to remove the `<span>` element.

Also, I think we have some issue with the margins/padding of the `<ul>` block element, or maybe the `<p>` element, the spacing after the list doesn't look good:

![Screenshot from 2022-02-18 23-10-57](https://user-images.githubusercontent.com/723451/154768253-4993d424-76b5-4eea-8b55-a00313306296.png)

Reply:
> While at it, can we change this link to be a regular one instead of an awkward-looking button-like thing? It should be enough to remove the `<span>` element.

OK.  I left it becaue, while it looks odd, it was like that originally.

> Also, I think we have some issue with the margins/padding of the `<ul>` block element, or maybe the `<p>` element, the spacing after the list doesn't look good:

I tried to fix that by adding spacing in the end, so it looked less odd, and it didn't work.  I don't know how to fix this, I think that it's the <p>.  Maybe with '\' at the end of the text it makes the list be included within the <p> but not sure if it will be done that way and parse the list as regular text and not a list... I suppose that it depends on each Markdown parser.

Reply:
No problem at all, someone else can give it a try to tweak the CSS for the list on a follow-up commit.

Reply:
Filed #101 to take care of this.